tags/tag-operational-resilience/charter.md

```diff
@@ -1 +1,83 @@
+
+## Introduction
+
+In today's digital landscape, the resilience and efficient operation of cloud-native systems are both a technical requirement and a business necessity. 
+As organizations rely on cloud native architectures to deliver services, the impact of downtime or performance degradation has grown significantly. 
+This Technical Advisory Group [(TAG)](https://github.com/cncf/toc/blob/main/governance/tag-governance.md) is established to address the complex challenges of maintaining system availability, performance, and reliability. By focusing on "Day 2 Operations," the period after a system is deployed, this TAG aims to provide the community with the principles, practices, and tools needed to build and operate robust and efficient cloud native services. The TAG will also assist the CNCF TOC with any project due diligences and necessary reviews within the scope of the TAG.
+
+## Mission Statement
+
+To define and advance practices and standards for building, operating, adopting  and managing resilient, observable, and efficient cloud native systems, applications, and architectures. 
+
+## Responsibilities & Deliverables
+
+### In-Scope
+
+This TAG covers the following sub-domains and topics within its scope:
+
+* **Observability:** Monitoring, logging, and tracing to gain deep insights into system behavior and health.
+* **Management:** The processes and tools essential for managing the complete lifecycle of cloud native applications.
+* **Business Continuity:** Strategies and practices to ensure that business operations can continue in the event of a disruption.
+* **Resource Optimization:** The efficient use of computational, storage, and network resources to balance performance and cost. 
+* **Cost Efficiency:** Methodologies for optimizing expenditures without compromising system reliability or performance. 
+* **Energy Efficiency:** Focusing on and providing guidance for reducing the energy consumption of cloud native services.
+* **Performance:** Ensuring the speed, responsiveness, and stability of cloud native applications under various load conditions.
+* **Troubleshooting:** The systematic process of identifying, diagnosing, and resolving issues within a distributed system.
+* **Testing:** Verifying the reliability and resilience of applications
+* **Reliability:** The ability of a system to perform its required functions under stated conditions for a specified period of time.
+* **Day 2 Ops:** A focus on the post-deployment activities required to keep a system running smoothly, including maintenance, scaling, updating and upgrading.
+* **Chaos Engineering:** The practice of experimenting on a system to build confidence in its capability to withstand turbulent and unexpected conditions.
+* **Conformance Programs:** Relevant ownership and development of conformance programs to help end users adapt and choose platforms and tools that adhere to established operational resilience standards
```

Comment:
Conformance programs usually have a certification element to them. That (certification) is out of scope correct?

Reply:
Yes certification itself always is in the hand of CNCF, as far as it is stated in the [conformance projects by CNCF](https://github.com/cncf/foundation/blob/main/conformance-programs.md) maybe we should just add the rel to this doc

Reply:
That's would be great if you could add that. Thank you!

Reply:
@mfahlandt add reference to conofrmance